server/persistence/kube/kube.go

```diff
@@ -58,7 +56,6 @@ func (c *Client) UpdateUserWorkspace(ctx context.Context, user string, workspace
 	}()
 
 	workspace.SetNamespace(c.workspacesNamespace)
-	log.Printf("cli.Update %v ", workspace)
```

Comment:
Same here.

Reply:
Added in 01e7d62c3d019785ff80f98b2bc24a25a46d210c